changelogs/fragments/7870-homebrew-cask-installed-detection.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - homebrew - detect already installed formulae and casks using json output from brew info (https://github.com/ansible-collections/community.general/issues/864).
```

Comment:
```suggestion
  - homebrew - detect already installed formulae and casks using JSON output from ``brew info`` (https://github.com/ansible-collections/community.general/issues/864).
```

Reply:
Thank you, I'd applied the suggestions and pushed the new version.